test/action_controller/serialization_test.rb

```diff
@@ -58,6 +58,17 @@ def render_array_using_implicit_serializer_and_meta
           end
         end
 
+        def render_array_using_implicit_serializer_and_links
+          with_adapter ActiveModel::Serializer::Adapter::JsonApi do
+
+            @profiles = [
+              Profile.new({ name: 'Name 1', description: 'Description 1', comments: 'Comments 1' })
+            ]
+
+            render json: @profiles, links: { self: "/profiles/1" }
```

Comment:
As discussed on [jsonapi PR](https://github.com/json-api/json-api/pull/834) it's best to use absolute links. I already updated this line.